tasks/Carter Racing.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+Decision Making
+A small racing team is at a crossroads.  They have been experiencing engine failures all season, and are down to their last dollars, and are in danger of losing their sponsors.  However, when they did not suffer engine failure, their car and team were very successful.  They are within the final hours before the last big race of the season.  If they place in the top 5, which is very plausible if they choose to race, they will secure a large sponsor, Goodstone Tires.  If they are struck with another engine failure, they will lose their current sponsors and will not be able to continue racing next season.  What do they do?
+
+# References
+Brittain, J. & Sitkin, S. (1986). Carter racing. Case Study.
+
+Link: https://execed.poole.ncsu.edu/wp-content/uploads/2020/06/Carter-Racing-A_B_C.pdf
+
+Kray, Laura J., and Adam D. Galinsky. "The debiasing effect of counterfactual mind-sets: Increasing the search for disconfirmatory information in group decisions." Organizational Behavior and Human Decision Processes 91.1 (2003): 69-81.
+
+Link: https://www.researchgate.net/profile/Laura-Kray-2/publication/222830434_The_debiasing_effect_of_counterfactual_mind-sets_Increasing_the_search_for_disconfirmatory_information_in_group_decisions/links/5aa56ac60f7e9badd9ab57df/The-debiasing-effect-of-counterfactual-mind-sets-Increasing-the-search-for-disconfirmatory-information-in-group-decisions.pdf
+
```

Comment:
I would add the pre-task prime source here: Galinsky, A. D., & Moskowitz, G. B. (2000). Counterfactuals as
behavioral primes: Priming the simulation heuristic and consideration of alternatives. Journal of Experimental Social Psychology,
36, 257–383